### ISSUE TITLE
[a11y]: DatePicker component has multiple accessibility issues

### ISSUE THREAD

**Issue by anish-mm-05** [carbon-design-system/carbon#20553]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.91.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker- Latest deployment

### Description

Found multiple accessibility issues on the Datepicker component
- Content is not within a landmark element
- The SVG element has no accessible name

<img width="1724" height="873" alt="Image" src="https://github.com/user-attachments/assets/a2f67f1d-9e9e-44e2-bb79-6a1cec1f6dd5" />

### Reproduction/example

https://stackblitz.com/run?file=src%2FApp.jsx

### Steps to reproduce

- Open date picker component calendar
- Scan the accessibility issues

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
Closing as a duplicate of https://github.com/carbon-design-system/carbon/issues/18543 "The SVG element has no accessible name"

and you can see a solution to implement on your end here for the "Content is not within a landmark element" https://github.com/carbon-design-system/carbon/issues/15766
